src/components/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <v-card flat id="name-translations-list">
+      <!-- List Headers -->
+      <v-row class="name-translation-title list-item__subtitle" no-gutters>
+        <v-col>
+          <h3>Name Translations</h3>
+        </v-col>
+      </v-row>
+
+      <!-- List Content -->
+      <v-row
+        class="names-translation-content"
+        v-for="(translation, index) in translationList"
+        :key="`name_translation_${index}`"
+        no-gutters>
+        <v-col class="text-truncate">
+         <span class="name-title">{{translation.value}}</span>
+
+        <br v-if="translation.action">
+        <v-chip v-if="translation.action === actionTypes.ADDED"
+          x-small label color="#1669BB" text-color="white">ADDED</v-chip>
+        <v-chip v-if="translation.action === actionTypes.EDITED"
+          x-small label color="#1669BB" text-color="white">CORRECTED</v-chip>
+        <v-chip v-if="translation.action === actionTypes.REMOVED"
+          x-small label color="#E0E0E0" text-color="grey darken-1">REMOVED</v-chip>
+        </v-col>
+
+        <!-- Actions Column -->
+        <v-col v-if="translation.action === actionTypes.EDITED || translation.action === actionTypes.REMOVED">
```

Comment:
What about ADDED? Is that handled via the else below (Edit button)?

BTW, check with Scott as I think he wants the dropdown edit button to be labeled "Correct".